src/Microsoft.Health.SqlServer/Features/Schema/Manager/Model/PaasSchema.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+
+namespace Microsoft.Health.SqlServer.Features.Schema.Manager.Model
+{
+    public class PaasSchema
```

Comment:
Once this is merged and version is upgraded on Jack's Sql branch @health-paas repo, we can delete the model added in there - https://microsofthealth.visualstudio.com/Health/_git/health-paas?version=GBpersonal%2Fjackliu%2Fupdate-rp-worker-to-support-sql&path=%2Fsrc%2FCommon%2FSqlServer.Api.Paas%2FFeatures%2FSchema%2FModel